index.bs

```diff
@@ -2950,13 +2966,19 @@ The following abstract operations support the implementation of the
 <div algorithm>
  <dfn abstract-op lt="ReadableStreamDefaultControllerEnqueue"
  id="readable-stream-default-controller-enqueue">ReadableStreamDefaultControllerEnqueue(|controller|,
- |chunk|)</dfn> performs the following steps:
+ |chunk|, |transferList|)</dfn> performs the following steps:
 
  1. If ! [$ReadableStreamDefaultControllerCanCloseOrEnqueue$](|controller|) is false, return.
```

Comment:
If `isOwning` is false, then we won't use `transferList`, so we should throw an exception if it is set to let the user know that they're holding it wrong.

Reply:
There is a theoretical backward compatibility issue, is the potential extra clarity gain worth it?

Reply:
Do you mean if existing code calls `controller.enqueue(chunk, { transfer: [ chunk ] })` it will suddenly start throwing an exception? That seems unlikely enough that I wouldn't worry about it. Even if someone was doing that, their code was arguably wrong since the second argument was never doing anything.

Reply:
I guess there is already a backward compatibility issue with just introducing the second parameter as a `StructuredSerializeOptions` instead of `any`.